src/pages/Provider/Provider.layout.vue

```diff
@@ -1,91 +1,40 @@
 <template>
   <div class="Provider">
     <provider-header
-      :logo-url="provider.logo_url"
-      :name="provider.commercial_name"
-      v-model:query-value="searchText"
-      @onSearch="getProductLists(id, searchText)"
-    />
-    <categories-slider
-      class="Provider__categories"
-      :categories="[]"
-      :categorySelected="''"
+      :logo-url="provider?.logo_url"
+      :name="provider?.commercial_name"
+      :query-value="searchText"
+      @update:query-value="$emit('update:searchText', $event)"
+      @onSearch="$emit('onSearch')"
     />
     <div class="Provider__container">
-      <suspense>
-        <template #default>
-          <provider-products :products="products" />
-        </template>
-        <template #fallback>
-          <base-loading />
-        </template>
-      </suspense>
+      <base-loading v-if="isLoading" />
```

Comment:
Why suspense was abandoned?

Reply:
In this case, the suspense wasn't working, since that component is not made any async call. Suspense continue working for the parent when it call the Providers the first time